app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ErrorsController < ApplicationController
+  def routing
```

Comment:
```routing```
I think we need to name this something different - really doesn't describe what it does